localstack/services/cloudformation/service_models.py

```diff
@@ -84,7 +84,16 @@ def add_defaults(resource, stack_name: str):
 
     def get_cfn_attribute(self, attribute_name):
         """Retrieve the given CF attribute for this resource"""
-        return self.props.get(attribute_name)
+        prop_candidate = self.props.get(attribute_name)
+        if prop_candidate:
+            return prop_candidate
+
+        if "." in attribute_name:
+            parts = attribute_name.split(".")
+            attribute = self.props
+            for part in parts:
+                attribute = attribute.get(part)
+            return attribute
```

Comment:
just temporary, since this whole method will soon be removed. this will enable resolving properties like `Outputs.NestedStackOutputA`